packages/grid/x-data-grid-pro/src/hooks/features/columnResize/gridColumnResizeApi.ts

```diff
@@ -0,0 +1,47 @@
+import { GridColDef } from '@mui/x-data-grid';
+
+export const DEFAULT_GRID_AUTOSIZE_OPTIONS = {
+  includeHeaders: false,
```

Comment:
I suspect this should be true. It makes more sense to me from a UX perspective. It's how Notion, Google Sheet, AG Grid behave by default.

```suggestion
  includeHeaders: true,
```